doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,105 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. Each
```

Comment:
Property values can be single values or they can include, `type` and `metadata` attributes as well. Property values which correspond to the standard JSON datatypes 'string', 'number', 'boolean' are rendered as regular JSON values. Property values which specify a custom type or include metadata are rendered as JSON objects with the following properties:

* type : A JSON string which represents the attribute type provided by the entity creator
* value: Represents the attribute value (encoded as JSON as per its datatype)
* A collection of properties one per metadata field associated to the attribute value